src/components/KLabel/KLabel.vue

```diff
@@ -75,23 +66,32 @@ const hasTooltip = computed((): boolean => !!(props.info || props.help || slots.
   line-height: var(--kui-line-height-30, $kui-line-height-30);
   margin-bottom: var(--kui-space-40, $kui-space-40);
 
-  .is-required {
-    color: var(--kui-color-text, $kui-color-text);
-    font-size: var(--kui-font-size-30, $kui-font-size-30);
-    font-weight: var(--kui-font-weight-semibold, $kui-font-weight-semibold);
-    margin-left: var(--kui-space-20, $kui-space-20);
+  &.is-required {
+    margin-left: calc($kLabelSpacingX + 6px); // 6px to compensate for the 6px width of the dot
+    position: relative;
+
+    &::before {
+      background-color: var(--kui-color-background-danger, $kui-color-background-danger);
+      border-radius: var(--kui-border-radius-circle, $kui-border-radius-circle);
+      bottom: calc(50% - 2px); // place the dot in the middle of the text
+      content: '';
+      height: 6px;
+      left: 0px;
+      margin-left: calc((-1 * $kLabelSpacingX) - 6px); // -6px to compensate for the 6px width
+      position: absolute;
+      width: 6px;
```

Comment:
suggestion: we use a static `6px` value a lot in this file. Assuming that's because they are all linked to a certain value, should we create a **local** SCSS variable at the top of the `style` block to define the `6px` value that can then be referenced in the rest of the file?